deep_translator/base.py

```diff
@@ -181,3 +202,14 @@ def _translate_batch(self, batch: List[str], **kwargs) -> List[str]:
             translated = self.translate(text, **kwargs)
             arr.append(translated)
         return arr
+
+    async def async_translate_batch(
+        self, batch: List[str], **kwargs
+    ) -> List[str]:
+        if not batch:
+            raise Exception("Enter your text list that you want to translate")
+        async with aiohttp.ClientSession() as session:
+            translation_tasks = [
+                self._async_translate(text, session) for text in batch
+            ]
+            return await asyncio.gather(*translation_tasks)
```

Comment:
I think you can get a ValueError here if translation_tasks is empty. You may want to add a check for that